cmd/format/format.go

```diff
@@ -243,7 +245,7 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 	return reader.Close()
 }
 
-// applyFormatters
+// applyFormatters.
```

Comment:
Lol this dot looks funny. Is the linter enforcing that we write actual sentences? Perhaps better to just remove this comment?

Reply:
Yeah I'll just remove it